src/main.rs

```diff
@@ -149,6 +175,44 @@ fn main() {
                     index_bytes_pipe!(buffer_path, prefix)
                 );
             }
+            Source::Ghcup(source) => {
```

Comment:
Would you please give an example of how data will be stored in filesystem, and bash commands to sync?